wiki/Beatmap/Difficulty/id.md

```diff
@@ -5,25 +5,24 @@ tags:
 
-Dalam ruang lingkup yang lebih spesifik, seberapa sukar suatu tingkat kesulitan turut dipengaruhi oleh hal-hal teknis seperti kompleksitas [pattern-pattern](/wiki/Beatmap/Pattern) yang ada dan [pengaturan tingkat kesulitan (*difficulty setting*)](/wiki/Beatmap_Editor/Song_Setup#difficulty) yang dipergunakan. Para pemain dapat bermain dengan memakai [modifier](/wiki/Game_modifier) untuk membuat suatu tingkat kesulitan menjadi lebih mudah atau lebih sukar untuk dimainkan.
+Hit object dari sebuah tingkat kesulitan menyusun sebuah [pola](/wiki/Beatmap/Pattern), yang juga bervariasi per mode permainan dan mengatur nada untuk tampilan dan nuansa keseluruhan tentang bagaimana kesulitan harus dimainkan. Permainan dari sebuah tingkat kesulitan beatmap juga dapat diubah oleh berbagai [pengaturan kesulitan](/wiki/Beatmap_Editor/Song_Setup#difficulty) yang diatur oleh pemilik beatmap atau dialihkan dengan [*game modifier*](/wiki/Game_modifier).
 
-Di samping itu, suatu berkas tingkat kesulitan juga mengandung berbagai data yang terkait dengan sampel-sampel [hitsound](/wiki/Beatmapping/Hitsound) yang dapat didengar di dalam permainan.
+Dari perspektif pendengar, sebuah tingkat kesulitan dapat menambah lagu dengan satu atau beberapa set sample audio, atau [*hitsound*](/wiki/Beatmapping/Hitsound) yang berfungsi sebagai masukan
 
 <!-- TODO: this description of visuals needs to be improved:
   - a separate section;
   - a very small paragraph on every key component, including those from Beatmap#overview;
 -->
 
-Dari sudut pandang visual, suatu berkas tingkat kesulitan dapat turut serta meliputi [berbagai komponen visual](/wiki/Beatmap) seperti [storyboard](/wiki/Storyboard) dan [skin](/wiki/Skinning) yang kemunculannya dapat diatur melalui menu [Visual Settings](/wiki/Visual_Settings). Di samping itu, terdapat pula berbagai informasi pendukung lainnya seperti waktu kemunculan [kiai time](/wiki/Kiai_time) dan [warna kombo](/wiki/Glossary/Combo_colour) yang ditampilkan.
+[Komponen visual](/wiki/Beatmap) dari sebuah tingkat kesulitan, berbeda dengan polanya, memberikan sebuah pengalaman tambahan selama bermain dan dapat dinonaktifkan di [*Visual setting*](/wiki/Visual_Settings) untuk sebagian besar. Sementara beberapa diantaranya, seperti [*storyboard*](/wiki/Storyboard) atau [*custom skin*](/wiki/Skinning), dari sebuah beatmap secara keseluruhan, sebagian besar komponen visual dapat bervariasi per tingkat kesulitan (sebagai contoh, bagian [kiai](/wiki/Kiai_time) dan [*combo colours*](/wiki/Glossary/Combo_colour)).
```

Comment:
```suggestion
[Komponen visual](/wiki/Beatmap) dari sebuah tingkat kesulitan berbeda dengan polanya, dan dapat memberikan pengalaman tambahan selama bermain. Komponen visual tersebut sebagian besar dapat dinonaktifkan melalui menu [*visual setting*](/wiki/Visual_Settings). Sementara itu, beberapa diantaranya seperti [*storyboard*](/wiki/Storyboard) atau [*custom skin*](/wiki/Skinning) dari sebuah beatmap secara keseluruhan, sebagian besar komponen visual tersebut dapat bervariasi pada masing-masing tingkat kesulitan (contohnya, bagian [kiai](/wiki/Kiai_time) dan [warna kombo](/wiki/Glossary/Combo_colour)).
```
can say that "combo colours" can be translated to "warna kombo" as stated on the original wiki id (https://osu.ppy.sh/wiki/en/Glossary/Combo_colour).